ideainspect.groovy

```diff
@@ -177,15 +175,14 @@ private OptionAccessor parseCli(configArgs) {
     h argName: 'help', longOpt: 'help', 'Show usage information and quit'
     l argName: 'level', longOpt: 'levels', args: Option.UNLIMITED_VALUES, valueSeparator: ',',
       'Levels to look for. Default: WARNING,ERROR'
-    s argName: 'file', longOpt: 'skip', args: Option.UNLIMITED_VALUES, valueSeparator: ',',
-      'Analysis result files to skip. For example "TodoComment" or "TodoComment.xml".'
-    sf argName: 'regex', longOpt: 'skipfile', args: Option.UNLIMITED_VALUES, valueSeparator: ',',
-       'Ignore issues affecting source files matching given regex. Example ".*/generated/.*".'
+    sc argName: 'string', longOpt: 'scope', args: 1,
+      'The name of the scope to be processed'
     t argName: 'dir', longOpt: 'resultdir', args: 1,
       'Target directory to place the IDEA inspection XML result files. Default: target/inspection-results'
     i argName: 'dir', longOpt: 'ideahome', args: 1,
       'IDEA or Android Studio installation home directory. Default: IDEA_HOME environment variable or "idea"'
     d argName: 'dir', longOpt: 'dir', args: 1, 'Limit IDEA inspection to this directory'
+    li argName: 'dir', longOpt: 'lib', args: 1, 'The directory contains the *.vmoptions file'
```

Comment:
I think you are looking for the `.properties` file, not `*.vmoptions`.

I'd rather replace it with the full path anyways.